ckanext/opendatani/templates/report/view.html

```diff
@@ -0,0 +1,26 @@
+{% extends "page.html" %}
+
+{% block title %}{{ 'Publisher Report' }} - {{ _('Reports') }} - {{ super() }}{% endblock %}
+
+
+{% block breadcrumb_content %}
+  {{ h.build_nav('home', _('Home')) }}
+{% endblock%}
+
+{% for csv_id in extra_vars %}
+  {% block primary_content_inner %}
+      <h1>{{ 'Publisher Report' }}</h1>
+      <p>{{ 'An up to date report instantly generated with publisher usage data.' }}</p>
+      <div class="panel panel-info">  
+          <div class="panel-body">
+            <form action="" method="POST">
+              <input type="submit" value="{% trans %}Refresh{% endtrans %}" class="btn btn-info pull-right" style="margin-left: 15px"/>
+            </form>
+          </div>
+      </div>
+      <div class="pull-right">
+          {{ _('Download') }}:
+          <a class="btn btn-primary" href="{{ '/tmp/' + csv_id }}">CSV</a>
```

Comment:
Can you explain why href is `'/tmp/' + csv_id `?

Reply:
This is due to where it's stored. This is redirecting the page to `http://ckan-dev:5000/report/test_org_1/tmp/<csv_id>`, which was the best way I found to directly access the files.

Reply:
So as I understand if we remove `tmp/` from storage path we can remove from URL as well correct? or at least let's change to smth reasonable like `r/` or  similar

Reply:
I changed it to `/publisher-reports/` for now. I could do something shorter, like `/r/`, but I thought maybe (for the sake of clarity) a more descriptive name would be best. At this point, that's just the storage location though. No one ever navigates to that URL manually anymore. What's called on button click is now `http://ckan-dev:5000/publisher-reports/<csv_id>`, which will automatically download the file (now without a redirect).

Reply:
Great